types/ember__object/evented.d.ts

```diff
@@ -36,21 +36,69 @@ interface Evented {
     off<Target>(
         name: string,
         target: Target,
-        method: (this: Target, ...args: any[]) => void
+        method: string | ((this: Target, ...args: any[]) => void)
     ): this;
-    off(name: string, method: (...args: any[]) => void): this;
+    off(name: string, method: string | ((...args: any[]) => void)): this;
     /**
      * Checks to see if object has any subscriptions for named event.
      */
     has(name: string): boolean;
 }
 declare const Evented: Mixin<Evented>;
 export default Evented;
+
 /**
  * Define a property as a function that should be executed when
  * a specified event or events are triggered.
+ *
+ * The actual implementation allows for any number of eventName strings followed by
+ * the function to be called. In TypeScript 4.0 this could be better described with:
+ *
+ *   type EventNames = string[];
+ *   type OnArgs = [...EventNames, (...args: any[]) => void];
+ *   export function on(...args: OnArgs): (...args: any[]) => void;
```

Comment:
Let’s move these to a `//`-style comment above this, instead of in the docstring: the docstring will show up for everyone who uses the type definition… which includes all JS users who are using VS Code, since it does automatic type acquisition in the background to power completions for the  language server. :scream: